msg/json/tl.json

```diff
@@ -62,165 +52,30 @@
 	"CONTROLS_IF_TOOLTIP_3": "Kung ang unang value ay true, gagawin ang first block ng statement. Kung hindi, kung ang second value ay true, gagawin ang second block ng statement.",
 	"CONTROLS_IF_TOOLTIP_4": "Kung ang first value ay true, gagawin ang first block ng statement. Kung hindi true ang second value, gagawin ang second block ng statement. Kung wala sa mga values ay true, gagawin ang last block ng statements.",
 	"CONTROLS_IF_MSG_IF": "kung",
-	"CONTROLS_IF_MSG_ELSEIF": "else if",
-	"CONTROLS_IF_MSG_ELSE": "else",
 	"CONTROLS_IF_IF_TOOLTIP": "Mag Add, remove o kaya mag reorder ng sections para maayos ang if block.",
 	"CONTROLS_IF_ELSEIF_TOOLTIP": "Mag dagdag ng condition sa if block.",
 	"CONTROLS_IF_ELSE_TOOLTIP": "Mag Add ng final, kunin lahat ng condition sa if block.",
-	"LOGIC_COMPARE_HELPURL": "http://en.wikipedia.org/wiki/Inequality_(mathematics)",
 	"LOGIC_COMPARE_TOOLTIP_EQ": "Nag babalik ng true kung ang pinasok ay parehong magkatumbas.",
 	"LOGIC_COMPARE_TOOLTIP_NEQ": "Return true if both inputs are not equal to each other.",
 	"LOGIC_COMPARE_TOOLTIP_LT": "Nag babalik ng true kung ang unang pinasok ay maliit kaysa sa pangalawang pinasok.",
 	"LOGIC_COMPARE_TOOLTIP_LTE": "Nag babalik ng true kung ang unang pinasok ay maliit sa o katumbas sa pangalawang pinasok.",
 	"LOGIC_COMPARE_TOOLTIP_GT": "Nagbabalik ng true kung ang unang pinasok ay mas malaki kaysa pangalawang pinasok.",
 	"LOGIC_COMPARE_TOOLTIP_GTE": "Nag babalik ng true kung ang unang pinasok ay mas malaki or katumbas ng pangalawang pinasok.",
-	"LOGIC_OPERATION_TOOLTIP_AND": "Return true if both inputs are true.",
 	"LOGIC_OPERATION_AND": "at",
-	"LOGIC_OPERATION_TOOLTIP_OR": "Return true if at least one of the inputs is true.",
 	"LOGIC_OPERATION_OR": "o",
-	"LOGIC_NEGATE_TITLE": "not %1",
-	"LOGIC_NEGATE_TOOLTIP": "Returns true if the input is false.  Returns false if the input is true.",
 	"LOGIC_BOOLEAN_TRUE": "tama",
 	"LOGIC_BOOLEAN_FALSE": "mali",
 	"LOGIC_BOOLEAN_TOOLTIP": "Nag babalik ng true or false.",
-	"LOGIC_NULL_HELPURL": "http://en.wikipedia.org/wiki/Nullable_type",
 	"LOGIC_NULL": "blangko",
-	"LOGIC_NULL_TOOLTIP": "Returns null.",
-	"LOGIC_TERNARY_HELPURL": "http://en.wikipedia.org/wiki/%3F:",
-	"LOGIC_TERNARY_CONDITION": "test",
 	"LOGIC_TERNARY_IF_TRUE": "kung tama",
 	"LOGIC_TERNARY_IF_FALSE": "kung mali",
-	"LOGIC_TERNARY_TOOLTIP": "Check the condition in 'test'. If the condition is true, returns the 'if true' value; otherwise returns the 'if false' value.",
```

Comment:
Was Tagalog always missing all these translations?

Reply:
From the previous PR: "This change also gets rid of a bunch of English that made it into Tagalog by accident, which gives it a chance to be retranslated. The Tagalog blocks will look the same, since by default we use English if there was no translation."